From af2f8efbc6bfc5937089e2c122ae1c3b99c016d4 Mon Sep 17 00:00:00 2001
From: Thor Thayer <thor.thayer@linux.intel.com>
Date: Fri, 8 Nov 2019 16:40:53 -0600
Subject: [PATCH 056/151] arm64: dts: agilex: Add SysMgr compatible

commit  18502d3cae549ea3e27902819e44875ced23eae1 from
https://github.com/altera-opensource/linux-socfpga.git
branch is socfpga-5.4.64-lts

[backport 'commit 5dfd391e10a0 ("arm64: dts: agilex: Add SysMgr compatible")']
from dinguyen/for_next_v5.5_socfpga_dts_v2

Add the System Manager compatible string to the sysmgr node
for accessing the System Manager driver.
Ethernet and EDAC drivers use this driver.

Signed-off-by: Thor Thayer <thor.thayer@linux.intel.com>
Signed-off-by: Dinh Nguyen <dinguyen@kernel.org>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
Integrated-by: Jun Zhang <jun.zhang@windriver.com>
---
 arch/arm64/boot/dts/intel/socfpga_agilex.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/intel/socfpga_agilex.dtsi b/arch/arm64/boot/dts/intel/socfpga_agilex.dtsi
index 59af428e763f..43b22dfca2bd 100644
--- a/arch/arm64/boot/dts/intel/socfpga_agilex.dtsi
+++ b/arch/arm64/boot/dts/intel/socfpga_agilex.dtsi
@@ -395,7 +395,7 @@
 		};
 
 		sysmgr: sysmgr@ffd12000 {
-			compatible = "altr,sys-mgr", "syscon";
+			compatible = "altr,sys-mgr-s10","altr,sys-mgr";
 			reg = <0xffd12000 0x500>;
 		};
 
-- 
2.26.1

